test/jasmine/tests/axes_test.js

```diff
@@ -2722,6 +2722,9 @@ describe('Test axes', function() {
 
     describe('calcTicks and tickText', function() {
         function mockCalc(ax) {
+            if(ax.minexponent === undefined) {
+                ax.minexponent = 3;
```

Comment:
We shouldn't need this here.

Reply:
There are other tests calling mockCalc which aren't setting minexponent. This causes them to fail.

Reply:
So maybe we should fix this either in the tests or here; but not both.

Reply:
Agreed! Fixed.